scripts/install/default-env-vars.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-export PANDIR=/var/panoptes
-export PANLOG=${PANDIR}/logs
-export POCS=${PANDIR}/POCS
-export PAWS=${PANDIR}/PAWS
-export PIAA=${PANDIR}/PIAA
```

Comment:
Do we not want/need PIAA?

Reply:
No, PIAA is going to change soon and I'm not sure we need it (at least for now).